index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Abdellah Portfolio</title>
+    <link rel="stylesheet" href="stylesheet.css">
+</head>
+<body>
+    <div class="firstSection">
+        <header>
+            <nav class="navbar">
+                <ul class="myNavLinks">
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </nav>
+        </header>
+        <main>
+            <p id="firstPara">Hey there.</p><p id="supportingPara">I'm Abdou.</p>
+            <p id="secondPara">I'm a software developer</p>
+            <p id="supportingText"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don't hestiate to contact me.</p>
+            <div class="social-media-links">
+                <a href="https://twitter.com/belkaiduus" target="_blank" rel="noopener"><img rel="noreferrer" src="img/Twitter.png" alt="Twitter Logo" ></a> 
+                <a href="https://linkedin.com/in/belkaidus" target="_blank" rel="noopener"><img rel="noreferrer" src="img/Linkedin.png" alt="Linkedin Logo" ></a>
+                <a href="https://github.com/mr-abdellah" target="_blank" rel="noopener"><img rel="noreferrer" src="img/GitHub.png" alt="GitHub Logo"></a>
+            </div>
```

Comment:
- [ ] Please the list of icons uses a `<ul>` tag to group them, and `<li>` per icon; each icon has a link (`<a>` tag)